cmd/go.mod

```diff
@@ -14,30 +14,62 @@ require (
 	google.golang.org/protobuf v1.28.1
 	k8s.io/client-go v0.25.2
 	k8s.io/klog/v2 v2.80.1
+	sigs.k8s.io/kpng/api v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/backends/ebpf v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/backends/iptables v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/backends/ipvs-as-sink v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/backends/nft v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/backends/userspacelin v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/backends/windows/kernelspace v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/backends/windows/userspace v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/client v0.0.0-20221014160047-e8543ceff80f
+	sigs.k8s.io/kpng/server v0.0.0-20221014160047-e8543ceff80f
 )
 
 require (
+	github.com/Microsoft/go-winio v0.4.17 // indirect
+	github.com/Microsoft/hcsshim v0.9.4 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/blang/semver/v4 v4.0.0 // indirect
+	github.com/cespare/xxhash v1.1.0 // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/cilium/ebpf v0.8.1 // indirect
```

Comment:
Yeah something is going on here we shouldn't have a dependency on cilium/ebpf here